RELEASE.md

```diff
@@ -55,6 +55,12 @@ Update version references with the following command:
 find .github/workflows/ -name '*.yaml' -o -name '*.yml' | xargs sed -i "s/uses: slsa-framework\/slsa-github-generator\/\.github\/actions\/\(.*\)@\(main\|v[0-9]\+\.[0-9]\+\.[0-9]\+\)/uses: slsa-framework\/slsa-github-generator\/.github\/actions\/\1@$BUILDER_TAG/"
 ```
 
+Likewise, update documentation with the following command:
+
+```shell
+find . -name "*.md" -exec sed -i "s/@v[0-9]\+\.[0-9]\+\.[0-9]\+/@$BUILDER_TAG/g" {} +
```

Comment:
Sometimes we may refer to old versions in the docs. For example in sections on "Known issues": https://github.com/slsa-framework/slsa-github-generator/tree/main/internal/builders/go#known-issues

Is there a way we can maybe get around the check and update the right version in those cases? Before we only updated references to the slsa-framework/slsa-github-generator workflows rather than all references to any semantic version.

Maybe we should wrap the versions we don't want to update in pseudo tags?

Reply:
I could modify the regex to only update tags prefixed with `uses: slsa-framework/slsa-github-generator`. So basically only updating tags within example workflow blocks.

Or, if you believe there could be cases where we'd want to reference the current version outside of code blocks, we could instead check that tags are prefixed with `@`. This would still detect all workflows and then within the broader text we could prefix the "current version" tags with `@` ("New features in @v1.2.3"), while leaving references to old versions alone ("Known issues with v1.2.1"). (Or visa-versa)

Or, if we want to use some markdown hacks, we can prefix old versions with `[]()` (i.e. `"Known issues with []()v1.2.1"`, which renders as "Known issues with []()v1.2.1". We can then filter out versions with this "prefix". For improved clarity, we can also add a descriptive identifier to the prefix: `[](skip_update)` or something of the sort (though it'd have to be the same everywhere, since we'd have to add it to the filter).

Reply:
I think for now just checking for the `uses: slsa-framework/slsa-github-generator` prefix would work.

BTW, another instance where we reference old versions in the docs in where we say which version a builder/generator was released as GA.
e.x. https://github.com/slsa-framework/slsa-github-generator#builders